### DISCUSSION TITLE
Segmentation Fault when trying to calculate Energy Density

### DISCUSSION THREAD

**Discussion by pagel-s** [Electrostatics/apbs#265] [Q&A]
I am trying to calculate the energy density for a protein (PDB-ID 6GK6, but have tried multiple others as well) and keep on getting
`Segmentation fault (core dumped)`

I am using the precompiled APBS v3.4.1 version on a Linux machine and python version 3.9.12
Charge density, however, is working without any problem and if have not had any other issues.

And I have attached the console output and my apbs.in file.
Any help would be greatly appreciated :)

[console.log.txt](https://github.com/Electrostatics/apbs/files/10272756/console.log.txt)
[apbs.txt](https://github.com/Electrostatics/apbs/files/10272780/apbs.txt)

**Comment by sobolevnrm** [comment]
Can you please also provide the input file and PQR file that caused this problem?
